src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2324,6 +2328,14 @@ def aks_trustedaccess_role_binding_create(cmd, client, resource_group_name, clus
         resource_type=CUSTOM_MGMT_AKS_PREVIEW,
         operation_group="trusted_access_role_bindings",
     )
+    try:
+        existedBinding = client.get(resource_group_name, cluster_name, role_binding_name)
+    except ResourceNotFoundError as ex:
+        print("TrustedAccess RoleBinding " + role_binding_name + " not existed, will create a new one!")
```

Comment:
Better use `logger.warning` instead.

print() would write output to stdout, while logger.warning would write output to stderr. In general, should not write anything to stdout by yourself.